parsers/ENTSOE.py

```diff
@@ -1000,6 +1018,67 @@ def parse_prices(
     return prices
 
 
+def get_exchange(
+    zone_key1: ZoneKey,
+    zone_key2: ZoneKey,
+    session: Session | None = None,
+    target_datetime: datetime | None = None,
+    logger: Logger = getLogger(__name__),
+    forecasted: bool = False,
+) -> list:
+    """Gets either the forcasted or measured net exchange between two zones."""
+    session = session or Session()
+
+    sorted_zone_keys = sorted([zone_key1, zone_key2])
+    key = ZoneKey("->".join(sorted_zone_keys))
+    if key in ENTSOE_EXCHANGE_DOMAIN_OVERRIDE:
+        domain1, domain2 = ENTSOE_EXCHANGE_DOMAIN_OVERRIDE[key]
+    else:
+        domain1 = ENTSOE_DOMAIN_MAPPINGS[zone_key1]
+        domain2 = ENTSOE_DOMAIN_MAPPINGS[zone_key2]
+    # Grab exchange
+    # Export
+    raw_exchange = (
+        query_exchange_forecast(domain1, domain2, session, target_datetime)
```

Comment:
Since the functions have the same parameters you could have some like this:
```
query_function =  query_exchange
if forecasted:
   query_function = query_exchange_forecast
...

query_function(domain1, domain2, session, target_datetime)
...
query_function(domain2, domain1, session, target_datetime)
```
This way you don't have to redo the if else.